code/obj/item/misc_weapons.dm

```diff
@@ -2130,6 +2130,62 @@ obj/item/whetstone
 #undef HALB_MED_STAMCOST
 #undef HALB_LIGHT_STAMCOST
 
+/obj/item/crashaxe
+	name = "crash axe"
+	desc = "A lightweight utility-survival axe designed to cut through the thin skin of pod hulls. Works just as well on bothersome crewmembers."
+	icon = 'icons/obj/items/weapons.dmi'
+	icon_state = "crashaxe"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	hitsound ='sound/impact_sounds/coconut_break.ogg' //shamelessly re-using this sound idea from my halberd
+
+	health = 5
+	tool_flags = TOOL_CUTTING | TOOL_CHOPPING
+
+	w_class = W_CLASS_SMALL
+	hit_type = DAMAGE_CUT
+	leaves_slash_wound = TRUE //yeah prolly
+	force = 14
+	stamina_damage = 1 //we WANT this to be low so crewmembers can run away easily
+	throwforce = 24 //probably needs tuning but these aren't too far divorced from tomahawks- encourage high-skill gameplay
+	throw_speed = 4
+	throw_range = 7
+
+	var/breakmode = TRUE
+	var/breaking_tool_flags = TOOL_CUTTING | TOOL_CHOPPING
+	var/prying_tool_flags = TOOL_CUTTING | TOOL_PRYING
+
+	HELP_MESSAGE_OVERRIDE({"Can be used as a crowbar. Throw the weapon (space+click) to inflict a half-second stun and two second disorient."})
+
+	New()
+		..()
+		src.setItemSpecial(/datum/item_special/tile_fling) //borrowing attributes from crowbar
+		BLOCK_SETUP(BLOCK_ROD)
+
+	// I refuse to add to the item/knife/butcher inheritance tree. I just won't do it.
+	throw_impact(atom/A, datum/thrown_thing/thr)
+		if(iscarbon(A))
+			var/mob/living/carbon/C = A
+
+			C.changeStatus("knockdown", 0.5 SECONDS) //give what's basically a defib stun; lets salvagers yoink weapons if they're savvy
+			C.changeStatus("disorient", 2 SECONDS) //give them getaway time
```

Comment:
Honestly for throwing a weapon you have to buy these numbers could definitely be longer, syndie daggers are 6 seconds although they don't have the additional utility or damage of the axe.